R/formatters.R

```diff
@@ -0,0 +1,276 @@
+# Formatters ####
```

Comment:
Should we move to `html_formatter` etc now or do ypu think it is better practice this way.
OK for me.

Reply:
You mean to deprecate the old functions like set_html_formatters? I would keep them deprecated until the next CRAN release and then defunct them. We had all these examples using the old functions. After defuncing them in a CRAN release I would remove them in dev.